UnitedIncome/components
# Caret Stroke

### Description
> Please provide a brief description of your changes below.

Adds an option to the caret icon that allows you to adjust the stroke size.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Modify the stroke size in the Storybook story.
